tests/console/docker_runc.pm

```diff
@@ -24,75 +24,43 @@ use strict;
 use warnings;
 
 sub run {
-    select_console("root-console");
+    my @runtimes = ("docker-runc");
+    push @runtimes, "runc" if !is_sle('=15');
 
-    install_docker_when_needed;
+    select_console("root-console");
```

Comment:
Since your test doesn't rely on any assert_screen calls, you could use `$self->select_serial_terminal;` instead

Reply:
good point thanks!